options/locale/locale_en-US.ini

```diff
@@ -3436,6 +3436,23 @@ owner.settings.cleanuprules.success.delete = Cleanup rule has been deleted.
 owner.settings.chef.title = Chef Registry
 owner.settings.chef.keypair = Generate key pair
 owner.settings.chef.keypair.description = A key pair is necessary to authenticate to the Chef registry. If you have generated a key pair before, generating a new key pair will discard the old key pair.
+upload.button = Upload
```

Comment:
This one seems like it could be under a top-level `upload` key instead, being useable everywhere.